patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -142,7 +142,18 @@
          this.field_78531_r.field_71424_I.func_76318_c("hand");
  
          if (this.field_175074_C)
-@@ -1498,6 +1514,13 @@
+@@ -1403,7 +1419,10 @@
+             GlStateManager.func_179128_n(5888);
+             GlStateManager.func_179094_E();
+             this.func_78468_a(0, p_180437_2_);
++            long start = System.nanoTime();
+             p_180437_1_.func_180447_b(p_180437_2_, p_180437_3_);
++            net.minecraftforge.client.CloudRenderer.time += System.nanoTime() - start;
++            net.minecraftforge.client.CloudRenderer.frames++;
```

Comment:
I think profiling is normally only enabled when the debug screen is open.
(`profiler.profilingEnabled == true`)
A patch to force profiling to be active might be nice if it's not difficult to do, it would be more general-purpose than the patch here.